sass/_position.scss

```diff
@@ -0,0 +1,57 @@
+// Position Utility
+// ================
+
+
+// Position
+// --------
+/// Shortcut for positioning an element,
+/// using the common TRBL syntax.
+/// @group position
+/// @param {list} $position [null] -
+///   A shorthand-syntax of positioning values
+///   using the common TRBL pattern,
+///   e.g. `absolute 0 0 3em auto` or `fixed 0 2%`.
+///   The position value (absolute, relative, etc)
+///   defaults to `absolute`.
+///   Any `null` values are applied to the side offsets.
+/// @output `top`, `right`, `bottom`, `left`, and `position` properties
+/// @example scss - stretch over the full viewport
+///   .overlay {
+///     @include position(fixed 0);
+///   }
+/// @example scss - top right offset (absolute is impied)
+///   .overlay {
+///     // could also use `null` for bottom & left values…
+///     @include position(2em 2em auto auto);
+///   }
+@mixin position(
```

Comment:
So in this branch, we basically will want to go through all of our projects and use `@include position` instead of `@include stretch`. Yes?

Reply:
right, I think this more clearly expresses the use-cases we've been using `stretch` for.